app/presenters/hyrax/work_show_presenter.rb

```diff
@@ -218,5 +216,12 @@ def member_of_authorized_parent_collections
         # member_of_collection_ids with current_ability access
         @member_of ||= Hyrax::CollectionMemberService.run(solr_document, current_ability).map(&:id)
       end
+
+      def members_include_viewable_image?
+        file_set_presenters.each do |presenter|
```

Comment:
What do you think about `file_set_presenters.any? { |presenter| presenter.image? && current_ability.can?(:read, presenter.id) }`? It took me a beat to figure out that the current code would detect a presenter meeting the conditions.